src/main/java/org/protege/editor/owl/server/http/handlers/CodeGenHandler.java

```diff
@@ -55,8 +55,31 @@ public void handleRequest(HttpServerExchange exchange) {
 		}
 	}
 
+	public static List<String> generateCodes(int seq, int cnt, String p,
```

Comment:
They're actually there but the alignment is way off: https://patch-diff.githubusercontent.com/raw/bdionne/protege-server/pull/3.diff

Let me see if I can fix it.